ci/licenses.sh

```diff
@@ -42,7 +42,7 @@ fi
 
 echo "Checking license count in licenses_flutter..."
 actualLicenseCount=`tail -n 1 flutter/ci/licenses_golden/licenses_flutter | tr -dc '0-9'`
-expectedLicenseCount=2 # When changing this number: Update the error message below as well describing all expected license types.
+expectedLicenseCount=3 # When changing this number: Update the error message below as well describing all expected license types.
```

Comment:
When changing this number, update the error message below as well describing all expected license types.
(Namely, add a line between lines 56 and 57 saying that bsdiff is covered by a different copyright, etc.)

Reply:
Done